Documentation/dev_workflow.md

```diff
@@ -0,0 +1,156 @@
+This document shows an example of development process for Xen and GRUB installed
```

Comment:
Tpically we start document with 1st level header

Reply:
Added.